module1/homework/grayscaleImages/compression.cpp

```diff
@@ -56,7 +56,10 @@ std::array<std::array<uint8_t, width>, height> decompressGrayscale(const std::ve
 void printMap(const std::array<std::array<uint8_t, width>, height>& inputMap) {
     for (const auto& mapRow : inputMap) {
         for (const auto& rowElement : mapRow) {
-            std::cout << rowElement;
+            if (rowElement < 33)
```

Comment:
33 is a magic number. `' '` is better. Variable named firstPrintableChar is even better. `iscntrl` or `isprint` from `<cctype>` would be the best :)